app/services/pipeline_report_service.rb

```diff
@@ -67,12 +67,13 @@ def call
   end
 
   def generate
-    pipeline_run, metadata = get_pipeline_status(@pipeline_run_id)
+    pipeline_run, metadata = get_pipeline_status(@pipeline_run)
```

Comment:
This makes no sense if you are sending in a pipeline_run now. We are returning a pipeline run and sending it as input?

Reply:
Removed pipeline_run as an output.